packages/flutter/lib/src/material/drawer.dart

```diff
@@ -228,6 +229,17 @@ class DrawerController extends StatefulWidget {
   /// By default, the color used is [Colors.black54]
   final Color scrimColor;
 
+  /// The horizontal width from the edge of the screen that will respond to a
+  /// drag gesture to open the drawer.
+  ///
+  /// By default, the value used is 20.0.
+  ///
+  /// If `MediaQuery.of(context).padding` is larger than the default drawer
```

Comment:
It's good to make this all a bit more specific here however `MediaQuery.of(context).padding`  isn't a value that can be compared with 20. And "properly acommodate..." doesn't really explain why we're doing this. In fact it's not clear to me why we factor in the padding. If the padding is only as big as the display notches, doesn't that just guarantee that the drag area will be obscured by the notch? 

I realize that you're just documenting the way things were written already.

Reply:
It seems that this is a bug. I think the implementation should be to set it to the padding + 20.0 to account for the notch. I filed https://github.com/flutter/flutter/issues/38313 to track the issue, but will attempt to resolve this in this PR as well.